### PR TITLE
Add form type for PCS

### PR DESCRIPTION
**Pull request by LJBabbage**
The classifier isn't set for PCS or ASHE, this seeds the data

### CODE REVIEW

File: models/bootstrap.go [hunk 1]
```diff
@@ -106,6 +106,8 @@ func bootstrapSQL() [104]string {
 		"INSERT INTO classifiertypeselector (classifiertypeselectorpk, id, surveyfk, classifiertypeselector) VALUES (32, 'f8bb1c52-e63a-11e7-80c1-9a214cf093ae', 16, 'COMMUNICATION_TEMPLATE');",
 		"INSERT INTO classifiertype (classifiertypepk, classifiertypeselectorfk, classifiertype) VALUES (42, 32, 'LEGAL_BASIS');",
 		"INSERT INTO classifiertype (classifiertypepk, classifiertypeselectorfk, classifiertype) VALUES (43, 32, 'REGION');",
+		"INSERT INTO classifiertype (classifiertypepk, classifiertypeselectorfk, classifiertype) VALUES (44, 21, 'FORM_TYPE');",
+		"INSERT INTO classifiertype (classifiertypepk, classifiertypeselectorfk, classifiertype) VALUES (44, 25, 'FORM_TYPE');",
```
Comment (VibhorMehra) [inline models/bootstrap.go]: `classifiertypepk` is a primary key and needs to be unique. `44` is used twice. Could we please change this in the row to `45`?

File: models/bootstrap.go [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package models
 
-func bootstrapSQL() [104]string {
-	sql := [104]string{
+func bootstrapSQL() [105]string {
+	sql := [105]string{
```
Comment (andrewmil) [inline models/bootstrap.go]: This should be 106 if you're adding two rows in?

Comment (pricem14pc) [reply models/bootstrap.go]: This does indeed require 106 to avoid models/bootstrap.go:110:3: array index 105 out of bounds [0:105]

Comment (pricem14pc) [reply models/bootstrap.go]: Successfully tested locally following changes.. schema created and populated as expected

**Pull request merged**